### PR TITLE
Fix Auhentication token works incorrect

### PR DESCRIPTION
**Pull request by ascrutae**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

### CODE REVIEW

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/GRPCChannel.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+public class GRPCChannel {
+    /**
+     * origin channel
+     */
+    private final ManagedChannel originChannel;
+    private final Channel channelWithInterceptors;
+
+    private GRPCChannel(String host, int port, List<ChannelBuilder> channelBuilders,
+        List<Interceptor> interceptors, int maxInboundMessageSize, boolean usePlaintext,
+        NameResolverProvider nameResolverFactory) throws Exception {
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/GRPCChannel.java]: I prefer  these three should be in a ChannelBuilder called StandardChannelBuilder
- int maxInboundMessageSize
- boolean usePlaintext
- NameResolverProvider nameResolverFactory

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/Interceptor.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.apache.skywalking.apm.agent.core.remote;
+
+import io.grpc.Channel;
+
+/**
+ * @author zhang xin
+ */
+public interface Interceptor {
+    Channel intercept(Channel channel);
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/Interceptor.java]: Based on Channel as input and output args, this class is a `ChannelDecorator`

Comment (wu-sheng) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/remote/Interceptor.java]: Method can also be named as `build`

**Comment by ascrutae** [comment]
Done!

**Pull request merged**